librosa/feature/spectral.py

```diff
@@ -1370,13 +1382,26 @@ def mfcc(y=None, sr=22050, S=None, n_mfcc=20, **kwargs):
     >>> plt.title('MFCC')
     >>> plt.tight_layout()
 
+    Compare different DCT bases
 
+    >>> m_slaney = librosa.feature.mfcc(y=y, sr=sr, dct_type=2)
+    >>> m_htk = librosa.feature.mfcc(y=y, sr=sr, dct_type=3)
+    >>> plt.figure(figsize=(10, 6))
+    >>> plt.subplot(2, 1, 1)
+    >>> librosa.display.specshow(m_slaney, x_axis='time')
+    >>> plt.title('RASTAMAT / Auditory toolbox (dct_type=2)')
+    >>> plt.colorbar()
+    >>> plt.subplot(2, 1, 2)
+    >>> librosa.display.specshow(m_htk, x_axis='time')
+    >>> plt.title('HTK-style (dct_type=3)')
+    >>> plt.colorbar()
+    >>> plt.tight_layout()
     """
 
     if S is None:
         S = power_to_db(melspectrogram(y=y, sr=sr, **kwargs))
 
-    return np.dot(filters.dct(n_mfcc, S.shape[0]), S)
+    return scipy.fftpack.dct(S, axis=0, type=dct_type, norm=norm)[:n_mfcc]
```

Comment:
norm is also a valid arg for `melspectrogram`

Reply:
This PR was merged over a year ago.  If this is still an issue, please open a new report.  Thanks!